### ISSUE TITLE
Could you help me to understand your loss funtion?

### ISSUE THREAD

**Issue by tomorrowing** [celarex/STS-CNN-STSbenchmark-Semantic-Similarity-Semantic-Textual-Similarity-CNN-HCTI-Tensorflow-Keras#1]
I'm confused about your loss funtion,could you explain it? Thanks for your help,
